docs/conversion/sakai_22_1-22_2_oracle_conversion.sql

```diff
@@ -3,17 +3,14 @@ create index TOOL_IDX on lti_content (TOOL_ID);
 -- End SAK-47582
 
 -- SAK-47837 start
-ALTER TABLE rbc_returned_criterion_out DROP FOREIGN KEY FK3sroha5yjh3cbvq0on02wf3fk;
```

Comment:
@vacercon shall we drop this constraint?

Reply:
I can see how the constraint FK3sroha5yjh3cbvq0on02wf3fk is created in MySQL in sakai_22_mysql_conversion.sql
But I do not find that in Oracle it is created anywhere, so it is not possible to drop it